sockeye/inference.py

```diff
@@ -1005,6 +1026,9 @@ def __init__(self,
         self.interpolation_func = self._get_interpolation_func(ensemble_mode)
         self.beam_size = self.models[0].beam_size
         self.batch_size = self.models[0].batch_size
+        # skip softmax for a single model, but not for an ensemble
+        self.skip_softmax = True if len(models) == 1 else False
```

Comment:
let's decide on `skip_softmax` in just a single location, probably in `load_models` and reuse it after.